articles/azure/azs-how-create-spn-powershell.md

```diff
@@ -208,8 +207,8 @@ Add-AzureRmEnvironment -Name "AzureStackUser" -ArmEndpoint $ArmEndpoint
 
 # Create your Azure Stack Admin (Subscription Owner) credentials
 # Note: This account CAN, but does not have to, be the same as your public Azure account
-$AzsUsernameAdmin = "<output form="azsusername" name="result" style="display: inline;">user@contoso.onmicrosoft.com</output>"
-$AzsUserPasswordAdmin = ConvertTo-SecureString -String '<output form="azspassword" name="result" style="display: inline;">Password123!</output>' -AsPlainText -Force
+$AzsUsernameAdmin = "admin@onboardingukcloud3gmail.onmicrosoft.com"
+$AzsUserPasswordAdmin = ConvertTo-SecureString -String 'Skyscape!!!' -AsPlainText -Force
```

Comment:
Can you just confirm that it is OK to have the password here?

Reply:
Wasn't the actual password, but I realised I broke the form so I've changed it anyway.